README.md

```diff
@@ -1,36 +1,41 @@
-# Yo Azure Data Studio - Extension and Customization Generator
+# Azure Data Studio - Extension and Customization Generator
 
-We have written a Yeoman generator to help get you started. We plan to add templates for most extension/customization types into this.
+[![Twitter Follow](https://img.shields.io/twitter/follow/azuredatastudio?style=social)](https://twitter.com/azuredatastudio)
+
+The Azure Data Studio Extension Generator is a Yeoman-based generator to help get you started with authoring extensions. There are 13 different possible templates for you to use.
 
 ## Install the Generator
```

Comment:
It might be valuable to name this section "install the generator and prerequisites", and mention the prereqs that need to be installed first?